c2corg_ui/templates/article/edit.html

```diff
@@ -0,0 +1,143 @@
+                <label><span translate>activities</span> *</label>
+                <div class="route-activities">
+                  <div ng-repeat="activity in activities" class="activity">
+                    <div>
+                      <div class="icon-{{activity}}" class="icon-" ng-click="editCtrl.pushToArray(article, 'activities', activity, $event)"
+                           ng-class="{'activity-selected' : article.activities.indexOf(activity) > -1}">
+                      </div>
+                      <p>{{activity | translate}}</p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+
+            </section>
+
+            <div class="form-group half half-group">
+
+              <div ng-class="{ 'has-success': article.article_type}">
+                <label><span translate>article_type</span> *</label>
+                <select class="form-control" ng-model="article.article_type" ng-options="rat as mainCtrl.translate(rat) for rat in ${article_types}"><option value=""></option></select>
+                <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="article.article_type"></span>
+              </div>
+
+              ## ARTICLE CATEGORIES
+              <div ng-class="{'has-success': article.article_categories}" ng-init="articleCategories = ${article_categories}">
+                <div class="ng-hide">{{article.articleC = article.articleCategories || articleC[1]}}</div>
+                <label><span translate>article_categories</span> *</label>
+                <select ng-options="type as mainCtrl.translate(type) for type in articleCategories" ng-model="article.article_categories" class="form-control"><option value=""></option></select>
+                <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="article.article_categories"></span>
+              </div>
+
+
+            </div>
+          </div>
+        </section>
+
+        ## MAP
+        <section class="section maps-info" ng-show="userCtrl.auth.isModerator()">
+          <h2 class="heading show-phone"><span translate>geolocation</span></h2>
+          <div class="content" id="edit-maps">
+            <app-map app-map-edit="true" app-map-draw-type="Polygon" class="map-edit"></app-map>
+          </div>
+        </section>
+
+        <section class="section">
+          <h2 class="heading show-phone" translate>description</h2>
+          <div class="content">
+
+            ## SUMMARY
+            <div id="summary-group" class="form-group full">
+              <label translate>summary</label>
+              <textarea name="summary" ng-model="article.locales[0].summary" class="form-control"></textarea>
+            </div>
+
+            ## DESCRIPTION
+            <div id="description-group" class="form-group full">
+              <label translate>description</label>
```

Comment:
``description`` is probably not a relevant field name for an article.
In v5 it's ``article core`` (really better??) See http://www.camptocamp.org/articles/edit